packages/http-client-csharp/generator/Microsoft.Generator.CSharp/src/Expressions/FormattableStringExpression.cs

```diff
@@ -23,21 +23,23 @@ public FormattableStringExpression(string format, IReadOnlyList<ValueExpression>
             Args = args;
         }
 
-        public FormattableStringExpression(string format, params ValueExpression[] args) : this(format, args as IReadOnlyList<ValueExpression>)
+        public FormattableStringExpression(FormattableString value)
         {
+            Value = value;
         }
```

Comment:
I think you are trying to combine `FormattableStringExpression` (this type) and a type with similar name `FormattableStringToExpression` together.
But I think they are representing different things.

This type is to construct a `$""` expression. In other words we could say this is constructing a `FormattableString` instance.
The other type `FormattableStringToExpression` is working as a shimming layer with the raw code writer so that you could make anything as expressions.
Since this is a new repo, I think it is very reasonable we do not keep the shimming layer.